jenkins/ci.suse.de/cloud-mkcloud6.yaml

```diff
@@ -21,6 +21,7 @@
         - 'cloud-mkcloud{version}-job-hyperv-{arch}'
         - 'cloud-mkcloud{version}-job-raid-{arch}'
         - 'cloud-mkcloud{version}-job-ssl-{arch}'
+        - 'cloud-mkcloud{version}-job-upgrade-{arch}'
```

Comment:
This fixes the issue introduced with commit 64f09b2ce3c74393fa01efd2a6bb667e65214581 so it should have been a separate commit. But I'm fine with it for now.